packages/tradershub/src/components/TradingAccountCard/TradingAccountCardComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button, ButtonProps, Text } from '@deriv/quill-design';
+
+type TTradingAccountCardContent = {
+    children: string;
+    title: string;
+};
+
+export const TradingAccountCardContent = ({ children, title }: TTradingAccountCardContent) => (
+    <div className='grow'>
+        <Text bold className='leading-200' size='sm'>
+            {title}
+        </Text>
+        <Text className='text-[12px] leading-100 w-5/6 lg:w-full'>{children}</Text>
+    </div>
+);
+
+type TTradingAccountCardButton = {
+    onSubmit?: ButtonProps['onClick'];
+};
+
+export const TradingAccountCardLightButton = ({ onSubmit }: TTradingAccountCardButton) => (
+    <Button
+        className='rounded-200 bg-solid-coral-100 text-solid-coral-700 enabled:hover:bg-solid-red-200'
```

Comment:
I'm not sure about this approach... Instead of using common `class` for buttons, we are creating new instance of button with needed classes in tailwind. In this case we have to create separate button components for all colors, sizes, etc... In my opinion we took a wrong turn :) If this is right approach and we have to use it, in this case this component must in `components` folder, not here...

Reply:
I agree with @sergei-deriv. Lets put it in components for now

Reply:
I do agree with you but I thought that with quill we can use it in the meantime but if it's agreed amongst us all then I shall do it. However, I'll create a seperate PR for this for all the common buttons we have currently in legacy.

Reply:
LGTM
Lets do this in separate PR then 
@mahdiyeh-deriv please help to create a card for this.